src/Input.test.jsx

```diff
@@ -0,0 +1,39 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import Input from './Input';
+
+describe('Input', () => {
+  const handleChange = jest.fn();
+  const handleClick = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const taskTitle = '안녕';
+
+  const renderInput = () => render(
+    <Input value={taskTitle} onChange={handleChange} onClick={handleClick} />,
```

Comment:
```suggestion
    <Input
      value={taskTitle}
      onChange={handleChange}
      onClick={handleClick}
    />
```

다음과 같이 작성하면 props 확인이 더 수월해질 것 같아요

Reply:
### 수정했습니다!
props파악을 좀더 쉽게하기위해 줄나눔을 해주었습니다